src/highdicom/sr/templates.py

```diff
@@ -1366,6 +2131,133 @@ def __init__(
+            raise ValueError(
+                'Sequence contains more than one SR Content Item.'
+            )
+        dataset = sequence[0]
+        if dataset.ValueType != ValueTypeValues.CONTAINER.value:
+            raise ValueError(
+                'Item #1 of sequence is not an appropropriate SR Content Item '
+                'because it does not have Value Type CONTAINER.'
+            )
+        if get_coded_name(dataset) != codes.DCM.MeasurementGroup:
+            raise ValueError(
+                'Item #1 of sequence is not an appropropriate SR Content Item '
+                'because it does not have name "Measurement Group".'
+            )
+        instance = ContentSequence.from_sequence(sequence)
+        instance.__class__ = cls
+        return instance
+
+    @property
+    def method(self) -> Union[CodedConcept, None]:
+        """Union[highdicom.sr.coding.CodedConcept, None]: measurement method"""
+        root_item = self[0]
+        matches = find_content_items(
+            root_item,
+            name=codes.SCT.MeasurementMethod,
+            value_type=ValueTypeValues.CODE
+        )
+        if len(matches) > 0:
+            return matches[0].value
+        return None
+
+    @property
+    def tracking_identifier(self) -> Union[str, None]:
+        """Union[str, None]: tracking identifier"""
+        root_item = self[0]
+        matches = find_content_items(
+            root_item,
+            name=codes.DCM.TrackingIdentifier,
+            value_type=ValueTypeValues.TEXT
+        )
+        if len(matches) > 0:
+            return matches[0].value
+        return None
+
+    @property
+    def tracking_uid(self) -> Union[str, None]:
+        """Union[str, None]: tracking unique identifier"""
+        root_item = self[0]
+        matches = find_content_items(
+            root_item,
+            name=codes.DCM.TrackingUniqueIdentifier,
+            value_type=ValueTypeValues.UIDREF
+        )
+        if len(matches) > 0:
+            return matches[0].value
+        return None
+
+    @property
+    def finding_type(self) -> Union[CodedConcept, None]:
+        """Union[highdicom.sr.coding.CodedConcept, None]: finding"""
+        root_item = self[0]
+        matches = find_content_items(
+            root_item,
+            name=codes.DCM.Finding,
+            value_type=ValueTypeValues.CODE
+        )
+        if len(matches) > 0:
+            return matches[0].value
+        return None
+
+    @property
+    def finding_sites(self) -> List[FindingSite]:
+        """List[highdicom.sr.content.FindingSite]: finding sites"""
+        root_item = self[0]
+        matches = find_content_items(
+            root_item,
+            name=codes.SCT.FindingSite,
+            value_type=ValueTypeValues.CODE
+        )
+        if len(matches) > 0:
+            return [FindingSite.from_dataset(m) for m in matches]
+        return None
+
+    @property
+    def measurements(self) -> List[NumContentItem]:
```

Comment:
Hmm...I generally agree and like the idea. However, we would then also need to change this on `PlanarROIMeasurementsAndQualitativeEvaluations` and `VolumetricROIMeasurementsAndQualitativeEvaluations`. The problem that I see with that is that instances of these classes are returned by the `get_planar_roi_measurement_groups()` and `get_volumetric_roi_measurement_groups()`, respectively.

At the moment one can do

```python
for group in get_planar_roi_measurement_groups():
    print(group.measurements)
```

This would then have to become

```python
for group in get_planar_roi_measurement_groups():
    print(group.get_measurements())
```
which is not so nice IMO. What do you think?

Reply:
Well I agree the second one looks a bit nicer but it's pretty marginal and the question is how often are you really going to just be iterating through all measurements found in a group? I guess it depends on what you are doing but I would think in most of my use cases (ML focused of course) I would be looking for a particular measurement to do something with and would not be interested in whatever other measurements may be there at that point. And the difference here becomes:

```python
for meas in group.measurements:
    if meas.name == codes.SCT.Diameter:
        diameter = meas.value
        break
```

vs 

```python
diameter = group.get_measurements(codes.SCT.Diameter)[0].value
```
which to me trumps the minor difference above.

Of course, we could also decide to have both?

Reply:
Or we expose the measurements as `Dict[pydicom.sr.coding.Code: List[highdicom.sr.NumContentItem]]`?

```python
diameter = group.measurements[codes.SCT.Diameter][0].value
```

Reply:
Hmm, that would have some advantages. But what if I wanted to filter measurements by finding site, tracking ID, or referenced image?

Reply:
Implemented via 0b5ae3de258abb2c7c17ed3433fe607c9ece0d25 and e1c07cd0fb18ba66deb57106bd62f1b7f48865e8 and